eng/automation/dotnet-install.sh

```diff
@@ -0,0 +1,1568 @@
+#!/usr/bin/env bash
```

Comment:
Why are we including this? We generally try to avoid having this script maintained in the repo if we can avoid it.

Reply:
It is for 10-min package docker solution. The base image of docker is 'Alpine 3.15.0' which is not supported in the download dotnet-install.sh. So need to update the download dotnet-install.sh a little bit. So I put the updated version in the repo

Reply:
Instead of installing dotnet we should start with one of the published dotnet docker containers which already contains dotnet.

Reply:
If the goal is to have the repo up and running in a docker container we should use the .devcontainer in the root of the repo as it should already be in that state. The devcontainer is exactly intended for getting started with our repo and working in github codespaces or vs code devspaces.

Reply:
Don't need this script. delete it.